addons/event/static/src/scss/event.scss

```diff
@@ -40,5 +40,11 @@
                 object-fit: cover;
             }
         }
+        .o_event_kanban_location {
+            max-width: 85%!important
```

Comment:
Don't use !important unless you really **really** have to.

Here you should be able to target the rule enough to make your max-width take precedence over the one from the o_text_overflow class.
With something like: `.o_event_kanban_location.o_text_overflow`

Reply:
One more thing, did you know about `calc`?
It's very handy for those kind of situations.

Here you want to take the full space "except one character left for the icon".
So you can do something like:
`max-width: calc(100% - 1rem);`

Reply:
nice trick :)